service/config.go

```diff
@@ -36,4 +36,13 @@ type Config struct {
 
 	// DisconnectChan will be closed by the server when the client disconnects
 	DisconnectChan chan<- struct{}
+
+	// Packages contains the packages that we are debugging.
```

Comment:
These three fields can be removed because we've added them to the `debugger.Config` as well, which we include within this config already anyway.